### PR TITLE
Improve absolute heuristic

### PR DESCRIPTION
**Pull request by Barth2501**
# Changes
absolute heuristic implemented and ready to work

### CODE REVIEW

**Comment by qchuchu** [comment]
Seems good to me

**Review by qchuchu — APPROVED** [review]

File: heuristics/absolut_heuristic.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+def sign_integer(value):
```
Comment (hinosxz) [inline heuristics/absolut_heuristic.py]: Already defined in `lib/util`, you can import it from there

File: heuristics/absolut_heuristic.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    if value > 0:
+        return 1
+    elif value < 0:
+        return -1
+    else:
+        return 0
+
+
+def distance_nb_coups(pos_1, pos_2):
```
Comment (hinosxz) [inline heuristics/absolut_heuristic.py]: Already defined in `heuristic1`

**Review by hinosxz — COMMENTED** [review]
Only two comments

File: heuristics/absolute_heuristic.py [hunk 3]
```diff
@@ -1,60 +1,66 @@
-from numpy import ndarray
+from lib.util import distance_nb_coups
 
-from lib.positions import get_human_positions, get_opponent_positions, get_our_positions
 
-from lib.util import manhattan_dist
+def absolute_heuristic(our_positions:dict, opponent_positions:dict, humans_positions:dict):
```
Comment (hinosxz) [inline heuristics/absolute_heuristic.py:4]: Prefer `Dict[...]` typing, but no blocker

**Review by hinosxz — APPROVED** [review]
LGTM 👍

**Pull request merged**